meta-client/src/test/java/org/apache/helix/metaclient/impl/zk/TestZkMetaClient.java

```diff
@@ -57,9 +57,8 @@ public class TestZkMetaClient {
   private static final String ZK_ADDR = "localhost:2183";
   private static final int DEFAULT_TIMEOUT_MS = 1000;
   private static final String ENTRY_STRING_VALUE = "test-value";
-  protected static final String TRANSACTION_TEST_KEY_PREFIX = "/sharding-key-0";
-  protected static String PARENT_PATH = TRANSACTION_TEST_KEY_PREFIX + "/RealmAwareZkClient";
-  protected static final String TEST_INVALID_PATH = TRANSACTION_TEST_KEY_PREFIX + "_invalid" + "/a/b/c";
+  protected static String PARENT_PATH = "/transactionOpTestPath";
+  protected static final String TEST_INVALID_PATH = "_invalid/a/b/c";
```

Comment:
Not an issue, just asking, why protected? Do you plan to use these two elsewhere in the future?

Reply:
Hmm good point, I guess not. Must've stuck since I translated a lot of the code from zkMulti test cases. I'll switch it to private - thanks!